.flake8

```diff
@@ -0,0 +1,8 @@
+[flake8]
```

Comment:
> The discussion of the original proposal ([#234](https://github.com/PyCQA/flake8/issues/234)) was closed as "too heated", subsequent feature and pull requests were marked as "spam" ([#1332](https://github.com/PyCQA/flake8/pull/1332), [#1421](https://github.com/PyCQA/flake8/issues/1421), [#1431](https://github.com/PyCQA/flake8/issues/1431), [#1447](https://github.com/PyCQA/flake8/issues/1447), [#1501](https://github.com/PyCQA/flake8/issues/1501)).

Reply:
flake8 refuses to support pyproject.toml (see below) and therefore we are using this config file.

> The discussion of the original proposal ([#234](https://github.com/PyCQA/flake8/issues/234)) was closed as "too heated", subsequent feature and pull requests were marked as "spam" ([#1332](https://github.com/PyCQA/flake8/pull/1332), [#1421](https://github.com/PyCQA/flake8/issues/1421), [#1431](https://github.com/PyCQA/flake8/issues/1431), [#1447](https://github.com/PyCQA/flake8/issues/1447), [#1501](https://github.com/PyCQA/flake8/issues/1501)).